### DISCUSSION TITLE
As a Forem admin, I want to be able to hit the 3 action dots for a user in the member list view as well as in the individual user view

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#17066] [Feature Requests]
Hi there!

As a Forem admin, I want to be able to hit the 3 action dots for a user in the member list view as well as in the individual user view giving me even quicker access to the actions menu for banishing spam accounts.

Who's with me? Yah!

**Comment by ellativity** [comment]
I think this might be of interest for @amywtlin @JennieOcken and @anujbhavsar96

**Comment by JennieOcken** [comment]
@leewynne Interesting.  What exact actions do you feel like you want would to be able to perform from the member list view?  Just banish or anything else?  Anything you would find handy to be able to do in bulk (across multiple members?

**Reply by leewynne** [reply]
Hmmmmm,

Banish
Reward credits
Export / download data
Send a mail

**Reply by JennieOcken** [reply]
Tell me more about send mail.  What would happen when you selected multiple members and hit this?

How worried are you about banishing the _wrong_ person?

**Reply by leewynne** [reply]
In retrospect I think I got overly excited there...

Bulk > Banish / reward credits / export data
Individually > Render existing email form so I can send a message for gratitude.
